modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -192,13 +192,14 @@ export default {
     },
     customOperationsByContext() {
       return this.customOperations.filter(op => {
-        if (op.context === 'update' && this.isUpdateOperation) {
```

Comment:
ℹ️ test update context separately from the `manuallyPublished` and `hasUrl` conditions